README.md

```diff
@@ -54,7 +54,7 @@ The `${HOMEBREW_PREFIX}/opt/socket_vmnet` directory is usually symlinked to `../
 
 Run the following command to start the daemon:
 ```bash
-sudo ${HOMEBREW_PREFIX}/opt/socket_vmnet/bin/socket_vmnet --vmnet-gateway=192.168.105.1 ${HOMEBREW_PREFIX}/var/run/socket_vmnet
+mkdir -p ${HOMEBREW_PREFIX}/var/run && sudo ${HOMEBREW_PREFIX}/opt/socket_vmnet/bin/socket_vmnet --vmnet-gateway=192.168.105.1 ${HOMEBREW_PREFIX}/var/run/socket_vmnet
```

Comment:
Could you split this to two lines

Reply:
And mkdir needs sudo too

Reply:
@AkihiroSuda, thanks for reviewing!
I just split it into two lines as you suggested.

However I didn't need sudo for `mkdir -p ${HOMEBREW_PREFIX}/var/run` (I'm on M1). `mkdir -p /var/run` needs `sudo` for sure.
If you are on an intel mac, could you double check if `${HOMEBREW_PREFIX}/var/run` needs `sudo`?

Reply:
> If you are on an intel mac, could you double check if ${HOMEBREW_PREFIX}/var/run needs sudo?

I doesn't for me, but it also already exists:

```console
$ ls -l $HOMEBREW_PREFIX/var/run
total 0
drwxr-xr-x  2 jan  admin  64  5 Mar  2021 dbus
drwxr-xr-x  3 jan  admin  96  8 Dec  2020 dnsmasq
```

I think Homebrew will always create `$HOMEBREW_PREFIX/var/homebrew` and have `.../var` be writable, so I think it should be good.

Reply:
You can also test by `mkdir -p ${HOMEBREW_PREFIX}/var/whatever` (when you already have `.../var`) to make sure that it's possible to create without `sudo`.